src/main/resources/io/jenkins/plugins/designlibrary/HeteroList/index.jelly

```diff
@@ -24,40 +24,34 @@ THE SOFTWARE.
 -->
 
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:s="/lib/samples" xmlns:l="/lib/layout">
-    <s:sample title="Hetero List Sample">
-      <p class="jdl-paragraph">
-        <!-- TODO I have no idea what heterogeneous means, should use a simpler word -->
-        ${%description}
-      </p>
+<j:jelly xmlns:j="jelly:core" xmlns:s="/lib/samples" xmlns:f="/lib/form">
+    <s:layout>
+      <s:section hideBorder="true">
+        <s:group>
+          <s:preview fullWidth="true">
+            <f:property field="config" />
```

Comment:
Can't seem to use this on a 13" mac, it overflows under the example content:
<img width="301" alt="image" src="https://github.com/user-attachments/assets/a3ceb8dd-98ca-4827-a236-eb03255f2f69" />

Reply:
Fixed now, that menu seems to have a different implementation to the regular ones.